docs/en/pid-eaa-issuance.rst

```diff
@@ -1022,7 +1148,7 @@ In the following table are listed HTTP Status Codes and related error codes that
       - The PID/(Q)EAA Issuer cannot fulfill the request because of missing parameters, invalid parameters or request malformed. (:rfc:`6749#section-5.2`).
     * - *400 Bad Request* [REQUIRED]
       - ``invalid_grant``
-      - The PID/(Q)EAA Issuer cannot fulfill the request because the provided authorization code is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client. (:rfc:`6749#section-5.2`).
+      - The PID/(Q)EAA Issuer cannot fulfill the request because the provided authorization code or Refresh Token is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client. (:rfc:`6749#section-5.2`).
```

Comment:
```suggestion
      - The PID/(Q)EAA Issuer cannot fulfill the request because the provided authorization code or Refresh Token is invalid, expired, revoked, or does not match the redirection URI used in the authorization request, or was issued to another client. (:rfc:`6749#section-5.2`).
```